Playwright-POM-PoC-ts/pages/home-page/home-page.ts

```diff
@@ -0,0 +1,20 @@
+import { Locator, Page } from "@playwright/test";
+import { BasePage } from "../base-page";
+import { HomePageLocators as Selectors } from "./home-page-locators";
+
+export class HomePage extends BasePage {
+  readonly url: string;
+  readonly pageTitle: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.url = "/";
```

Comment:
U can also use `super(url)` if U do not plan to define other `url` than for BasePage 🙂